### PR TITLE
REDSHOP-5756: [Refactor] - Tags System - Refactoring Code - Template section: Clickatell SMS

### PR DESCRIPTION
**Pull request by hoang-tran-redweb**
refactor session clicktellsms

### CODE REVIEW

File: component/site/models/checkout.php [hunk 1]
```diff
@@ -1104,6 +1100,10 @@ public function orderplace()
             return false;
         }
 
+        if ($row->order_status == Redshop::getConfig()->get('CLICKATELL_ORDER_STATUS')) {
+            RedshopHelperClickatell::clickatellSMS($orderId);
```
Comment (anhredweb) [inline component/site/models/checkout.php]: @hoang-tran-redweb From now on, please help to add \ before class name as best pratice.
Thanks.
H.A

File: libraries/redshop/tags/sections/clicktellsms.php [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+	/**
+	 * Init
+	 *
+	 * @return  void
+	 *
+	 * @since __DEPLOY_VERSION__
+	 */
+	public function init()
+	{
```
Comment (anhredweb) [inline libraries/redshop/tags/sections/clicktellsms.php]: @hoang-tran-redweb please setup PSR-12 on PHPStorm and format code.
Thanks,
H.A

**Review by anhredweb — CHANGES_REQUESTED** [review]
@hoang-tran-redweb : please view my comments and help to update.
Thanks,
H.A

File: component/site/models/checkout.php (hunk 1 above)
Comment (hoang-tran-redweb) [reply component/site/models/checkout.php]: @anhredweb I fixed.

File: libraries/redshop/tags/sections/clicktellsms.php (hunk 2 above)
Comment (hoang-tran-redweb) [reply libraries/redshop/tags/sections/clicktellsms.php]: @anhredweb I setup PSR-12 and format code, please check

**Review by anhredweb — APPROVED** [review]

**Pull request merged**